cluster/examples/kubernetes/ceph/admission-controller/deployment/roles.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rook-admission-controller-sa
+  namespace: rook-ceph
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-admission-controller-role
+rules:
+- apiGroups: ["","ceph.rook.io"]
+  resources: ["pods","cephclusters"]
+  verbs: ["get", "watch", "list"]
+
```

Comment:
unnecessary empty line

Reply:
`Please update the doc as well as the pending release note file.`
Could you please point me towards these files ?